.gitignore

```diff
@@ -7,3 +7,4 @@ testing
 up-proxy
 !cmd/up-proxy
 dist
+.idea
```

Comment:
We should remove this as `.idea` isn't specific to `up(1)` itself, you can add `.idea` to your global `.gitignore`. See https://help.github.com/articles/ignoring-files/ for more details